### ISSUE TITLE
'No authority' when trying to activate SolisCloud API

### ISSUE THREAD

**Issue by YanisKyr** [hultenvp/solis-sensor#219]
I am getting this error when trying to activate the SolisCloud API. Any ideas what's blocking me?

https://i.imgur.com/yTTG9Pi.jpg

**Comment by hultenvp** [comment]
No, I don't. Perhaps if you open a topic in discussion others might be able to help.
Converting to discussion
